cmd/root/alias.go

```diff
@@ -0,0 +1,200 @@
+package root
+
+import (
+	"fmt"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/docker/cagent/pkg/shortcuts"
+	"github.com/docker/cagent/pkg/telemetry"
+)
+
+// NewAliasCmd creates a new alias command for managing aliases
+func NewAliasCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "alias [<alias-name> <agent-path>]",
+		Short: "Manage aliases for agents",
+		Long: `Create and manage aliases for agent configurations or catalog references.
+
+When called without arguments, lists all registered aliases.`,
+		Example: `  # List all registered aliases (default)
+  cagent alias
+
+  # Create an alias for a catalog agent
+  cagent alias code agentcatalog/notion-expert
+
+  # Create an alias for a local agent file
+  cagent alias myagent ~/myagent.yaml
+
+  # List all registered aliases
+  cagent alias list
+
+  # Remove an alias
+  cagent alias remove code`,
+		Args: cobra.RangeArgs(0, 2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// Default to list when no arguments provided
+			if len(args) == 0 {
+				telemetry.TrackCommand("alias", []string{"list"})
+				return listAliases()
+			}
+
+			// Create new alias with 2 arguments
+			if len(args) == 2 {
+				telemetry.TrackCommand("alias", args)
+				return createAlias(args[0], args[1])
+			}
+
+			// Single argument is invalid in this context
+			return fmt.Errorf("invalid arguments: use 'alias list' or 'alias <name> <agent-path>'")
+		},
+	}
+
+	cmd.AddCommand(newAliasListCmd())
+	cmd.AddCommand(newAliasRemoveCmd())
+
+	return cmd
+}
+
+// newAliasListCmd creates the list subcommand
+func newAliasListCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "list",
+		Short: "List all registered aliases",
+		Args:  cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			telemetry.TrackCommand("alias", []string{"list"})
+			return listAliases()
+		},
+	}
+}
+
+// newAliasRemoveCmd creates the remove subcommand
+func newAliasRemoveCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "remove <alias-name>",
```

Comment:
Can you add `rm` as an alias to this command?

Reply:
done